lib/msal-common/test/authority/Authority.spec.ts

```diff
@@ -911,7 +911,7 @@ describe("Authority.ts Class Unit Tests", () => {
                 });
             });
 
-            it("throws untrustedAuthority error if host is not part of knownAuthorities, cloudDiscoveryMetadata and instance discovery network call doesn't return metadata", (done) => {
+            it("throws untrustedAuthority error if host is not part of knownAuthorities, cloudDiscoveryMetadata and instance discovery network call doesn't return metadata, and the error returned from the network is 'invalid_instance'", (done) => {
```

Comment:
Can we have the "opposite" test as well? I.e. "if the error returned from AAD is not `invalid_instance` then ... "